### ISSUE TITLE
CSS not loading properly when navigating back and forth from the Alerts page

### ISSUE THREAD

**Issue by dillu24** [SimplyVC/panic#346]
### Rationale

Sometimes when navigating the installer using the PANIC UI navigation buttons the CSS is not rendered properly, especially when navigating back and fort from the alerts page. This is also breaking the toasts, because it was noted that if the css is broken then the toast are not raised. 

### For ticket closure

The aim of this ticket is to identify the problem and solve it

**Comment by dillu24** [comment]
See screenshot below:

![image](https://user-images.githubusercontent.com/23262259/187028674-3cd7f6a3-f2f5-4a80-ab39-444dc7cda4c3.png)

This was obtained when adding a new Substrate subchain from the CRUD journey, just after adding a substrate node.

**Comment by itsciccio** [comment]
It should be noted that this occurs throughout all installer steps... what triggers it is the mystery. However we think it is an issue with the new router.

**Comment by itsciccio** [comment]
Could be that the css file for the respective page is not being loaded

**Comment by zimaah** [comment]
TLDR: The **raiseToast()** function was appending the "toast" as child of **svc-app** tag and this was messing up the styles, since **svc-app** needs the "content container" as **first child** and not the **svc-toast** element. The correct approach is to append the **svc-toast** as child of **body** element.
